### DISCUSSION TITLE
Wondering why some properties are changed

### DISCUSSION THREAD

**Discussion by zaydek** [facebook/stylex#262] [Q&A] [answered]
For example, I noticed at the time of writing this that some properties like block-size are changed to height. Is this a compatibility concern?

Thanks.

**Comment by predaytor** [comment]
The same for `inline-size`, which is converted to `width`

**Comment by nmn** [comment, marked as answer]
Yes, the properties are changed for compatibility. Currently thinking about how to make this more configurable, but LTR and RTL layouts are fully supported. Vertical writing modes may not work as expected at the moment.

**Reply by zaydek** [reply]
OK no worries. I figured as much. I just want to make sure LTR and RTL just work.
